Signal/src/ViewControllers/ConversationView/ConversationViewModel.m

```diff
@@ -618,6 +624,13 @@ - (void)updateViewWithOldItemIdList:(NSArray<NSString *> *)oldItemIdList
     OWSAssertDebug(oldItemIdList);
     OWSAssertDebug(updatedItemSet);
 
+    if (!self.delegate.isObservingVMUpdates) {
+        OWSFailDebug(@"Skipping VM update.");
+        // We fire this event, but it will be ignored.
+        [self.delegate conversationViewModelDidUpdate:ConversationUpdate.minorUpdate];
```

Comment:
Calculating the incremental VM update diffs is non-trival, we can skip this work.